### PR TITLE
Examples and fancy tools

### PR DESCRIPTION
**Pull request by sblack4**

### CODE REVIEW

File: examples/basic/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | n/a |
```
Comment (smiller171) [inline examples/basic/README.md:13]: Need a version specified

File: examples/basic/main.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  version = "1.0.0"
+
+  names = [local.name, local.env, local.namespace]
+
+  tags = merge({
+    "Env"       = local.env,
+    "Namespace" = local.namespace,
+    "Owner"     = local.owner
+  }, local.extra_tags)
+}
```
Comment (smiller171) [inline examples/basic/main.tf:27]: Don't think we need the tags module in the example

File: examples/external-logging/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+No provider.
```
Comment (smiller171) [inline examples/external-logging/README.md:12]: This isn't right...

**Review by smiller171 — CHANGES_REQUESTED** [review]

File: examples/basic/README.md (hunk 1 above)
Comment (sblack4) [reply examples/basic/README.md:13]: I'm not wasting time perfecting examples

File: examples/basic/main.tf (hunk 2 above)
Comment (sblack4) [reply examples/basic/main.tf:27]: Yeah we do - we're demonstrating the Rhythmic way of doing things and getting a few downloads on our registry page

File: examples/external-logging/README.md (hunk 3 above)
Comment (sblack4) [reply examples/external-logging/README.md:12]: 🤷

File: examples/basic/README.md (hunk 1 above)
Comment (smiller171) [reply examples/basic/README.md:13]: 🙄

File: examples/basic/main.tf (hunk 2 above)
Comment (smiller171) [reply examples/basic/main.tf:27]: fair enough

**Review by smiller171 — APPROVED** [review]

**Pull request merged**